clients/src/main/java/org/apache/kafka/common/requests/OffsetFetchResponse.java

```diff
@@ -39,13 +42,13 @@
     private static final String PARTITION_KEY_NAME = "partition";
     private static final String COMMIT_OFFSET_KEY_NAME = "offset";
     private static final String METADATA_KEY_NAME = "metadata";
-    private static final String ERROR_CODE_KEY_NAME = "error_code";
+    private static final String PARTITION_ERROR_CODE_KEY_NAME = "error_code";
```

Comment:
nit: not really sure we need two separate constants even though they are separate fields in the struct.